src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: " + FigureName.CIRCLE.name()
+                + ", area: " + getArea()
+                + ", sq.units, radius: " + radius
+                + ", units, color: " + getColor());
```

Comment:
[To get a string representation of an enum use `name()`](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)
```suggestion
                + ", units, color: " + getColor().name());
```